flight/Modules/VtolPathFollower/vtol_follower_control.c

```diff
@@ -426,7 +308,7 @@ int32_t vtol_follower_control_endpoint(const float dT, const float *hold_pos_ned
 int32_t vtol_follower_control_land(const float dT, const float *hold_pos_ned,
 	bool *landed)
 {
-	return vtol_follower_control_simple(dT, hold_pos_ned, false, true);
+	return vtol_follower_control_impl(dT, hold_pos_ned, true, true);
```

Comment:
Ah. So this is why @pug398 was having issues with landing. That makes sense, I couldn't really understand that.

I wonder if we should actually apply this fix to master and make a hotfix release. Having RTH broken is pretty bad...